artifactory/services/utils/aqlquerybuilder.go

```diff
@@ -245,6 +245,7 @@ func buildReleaseBundleQuery(params *ArtifactoryCommonParams) (string, error) {
 // Creates a list of basic required return fields. The list will include the sortBy field if needed.
 // If requiredArtifactProps is NONE or 'includePropertiesInAqlForSpec' return false,
 // "property" field won't be included due to a limitation in the AQL implementation in Artifactory.
+// Alternatively, send two separate AQL, one with 'sort'/'limit' and the other with 'property' fields.
```

Comment:
I think this whole comment requires rephrasing. It is not very clear. The last sentence, starting with "Alternatively", is also not very clear. It hard to understand what "alternatively" mean in this context.

Reply:
Removed.